public/js/monitor.js

```diff
@@ -57,59 +57,79 @@ function isValidEmail(val) {
   return re.test(String(val).toLowerCase());
 }
 
-function addClass(selector, className){
+function addClass(selector, className) {
   const el = document.querySelector(selector);
   if(el){
     el.classList.add(className);
   }
 }
 
-function removeClass(selector, className){
+function removeClass(selector, className) {
   const el = document.querySelector(selector);
   if(el){
     el.classList.remove(className);
   }
 }
 
-function removeInvalidMessage(){
+function removeInvalidMessage() {
   removeClass("#invalid-email-message", "show");
   removeClass(".input-group","invalid");
 }
 
-function addInvalidMessage(){
+function addInvalidMessage() {
   addClass("#invalid-email-message", "show");
   addClass(".input-group", "invalid");
 }
 
 function enableBtnIfEmailValid(e) {
+  removeInvalidMessage();
   const emailBtn = document.getElementById("submit-email");
   if (isValidEmail(e.target.value)) {
       emailBtn.disabled = false;
       removeInvalidMessage();
   } else {
       emailBtn.disabled = true;
+      if(e.keyCode === 13) {
+        addInvalidMessage();
+      }
   }
+
 }
 
 function enableBtnIfInputEmpty(e) {
   const emailBtn = document.getElementById("submit-email");
-  if(e.target.value === ""){
+  if(e.target.value === "") {
       emailBtn.disabled = false;
       removeInvalidMessage();
   } else {
       enableBtnIfEmailValid(e);
   }
 }
 
-function showFalseDoor(){
+function closeByEscape(e) {
+  if(e.keyCode === 27 || e.keyCode === 13) {
```

Comment:
Per https://github.com/mozilla/blurts-server/pull/209/files#r197845073 above

> `keyCode` is deprecated, let's do `e.code === "Escape" || e.code === "Enter"`

Reply:
so this is weird. e.code works just fine with "Escape" but does absolutely nothing with "Enter" ?